project/Dependencies.scala

```diff
@@ -15,14 +15,15 @@ import sbt._
 object Dependencies {
   val resolvers = Seq(
     ("Snowplow Analytics Maven releases repo" at "http://maven.snplow.com/releases/").withAllowInsecureProtocol(true),
-    "Snowplow Bintray Maven repo"            at "https://snowplow.bintray.com/snowplow-maven"
+    "Snowplow Bintray Maven repo"            at "https://snowplow.bintray.com/snowplow-maven",
```

Comment:
Is the bintray repo still used for anything, now that you have added the twitter repo?

Reply:
good point, I'll remove it